index.js

```diff
@@ -16,43 +52,57 @@ module.exports = {
       name: options.name,
 
       defaultConfig: {
-        configuredServices: {
-          bugsnag: {
-            url: 'http://notify.bugsnag.com/deploy'
+        configuredServices: function(context) {
+          return {
+            bugsnag: {
+              url: 'http://notify.bugsnag.com/deploy',
+              method: 'POST',
+              headers: {},
+              body: function() {
+                var apiKey = this.apiKey;
+
+                if (!apiKey) { return; }
```

Comment:
But then the way to activate Bugsnag would be to define the API key which think is a bit implicit. I think it's nicer to separate configuration and activation and use the same behavior for all services.